video-img.py

```diff
@@ -0,0 +1,65 @@
+import cv2
+import time
+import numpy as np
+import os
+
```

Comment:
import Path from [pathlib](https://docs.python.org/3/library/pathlib.html). It helps dealing with path